substrate/frame/support/src/traits/misc.rs

```diff
@@ -916,30 +916,30 @@ pub trait IsInherent<Extrinsic> {
 /// An extrinsic on which we can get access to call.
 pub trait ExtrinsicCall: sp_runtime::traits::Extrinsic {
 	/// Get the call of the extrinsic.
-	fn call(&self) -> &Self::Call;
+	fn call(&self) -> Self::Call;
```

Comment:
We should probably return `Result` here to make it more obvious that this maybe includes decoding or maybe any better approach.